en/docs/reference/connectors/salesforce-pubsub-connector/sf-pubsub-connector-reference.md

```diff
@@ -0,0 +1,104 @@
+# Salesforce Pub/Sub Connector Reference
+
+
+The following operations allow you to work with the Salesforce Pub/Sub Connector. Click an operation name to see parameter details and samples on how to use it.
+
+
+### Connection configuration
+
+
+The Salesforce Pub/Sub Connector uses  Basic Authentication or Authorization headers. For more information on authentication, go to [Setting up Salesforce Pub/Sub Connector Connection]({{base_path}}/reference/connectors/salesforce-pubsub-connector/sf-pubsub-configuration/).
+
+
+??? note "Salesforce Pub/Sub Connection"
+
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
```

Comment:
Can we give possible values/ sample for this.

Reply:
Added the example